tests/main/interfaces-ssh-keys/task.yaml

```diff
@@ -21,12 +22,24 @@ prepare: |
     echo "testkey" > "$TESTKEY"
     echo "testkey.pub" > "$TESTKEY".pub
 
+    if [ -d "$CONFDIR" ]; then
+        cp -rf "$CONFDIR" "$CONFDIR".old
```

Comment:
I think you want to `cp -a` instead but in reality you just should not copy it at all. Instead please just mkdir the directory, touch the file and then in the restore rm the file and rmdir the directory and ignore the error.

Reply:
:+1:  Updated